Add `UpdateTimeZoneFromLongitude` to `ACesiumSunSky` (#1590)
CesiumGS/cesium-unreal: CHANGES.md

##### Additions :tada:

- Added `CesiumUrlTemplateRasterOverlay`, allowing a raster overlay to be added using tiles requested based on a specified URL template.

> - Added `UpdateTimeZoneFromLongitude` method to `ACesiumSunSky` to naively update the `ACesiumSunSky`'s `TimeZone` property based on a given longitude.

Reviewer: ```suggestion
- Added `AdjustTimeZoneForLongitude` to `ACesiumSunSky` to set a reasonable `TimeZone` value at the given longitude.
```

> - The "Place Georeference Origin Here" button on `ACesiumGeoreference` will now update the time zone of any `ACesiumSunSky` instances using that georeference based on the new origin's longitude.

Reviewer: ```suggestion
- The "Place Georeference Origin Here" button on `ACesiumGeoreference` will now adjust the time zone of the `ACesiumSunSky`(s) that reference it, based on the new origin's longitude. This improves user experience when moving the origin to locations where it is currently nighttime.
```

Author: It's "when moving the origin to locations where it would be nighttime in the current time zone," right?

Reviewer: Yeah that's better wording for what I was trying to say. :')
